server.go

```diff
@@ -248,7 +248,7 @@ func (server *Server) startProviders() {
 		log.Infof("Starting provider %v %s", reflect.TypeOf(provider), jsonConf)
 		currentProvider := provider
 		safe.Go(func() {
-			err := currentProvider.Provide(server.configurationChan, &server.routinesPool)
+			err := currentProvider.Provide(server.configurationChan, &server.routinesPool, server.globalConfiguration.Constraints.Get().([]types.Constraint))
```

Comment:
Isn't `server.globalConfiguration.Constraints` sufficient instead of `server.globalConfiguration.Constraints.Get().([]types.Constraint)` ?